android/main/java/com/couchbase/lite/internal/CouchbaseLiteInternal.java

```diff
@@ -143,7 +141,7 @@ public static Context getContext() {
     @NonNull
     public static String makeDbPath(@Nullable String rootDir) {
         requireInit("Can't create DB path");
-        return verifyDir((rootDir != null) ? new File(rootDir) : new File(getContext().getFilesDir(), DB_DIR_NAME));
+        return verifyDir((rootDir != null) ? new File(rootDir) : getContext().getFilesDir());
```

Comment:
The code before fixing and the code after fix could produce different db path. Is this backward compat with 2.8.0 (e.g. app that is used 2.8.0 without using 2.7.0 before)?

Reply:
It is *not* backwards compatible with 2.8.0.  It *is* compatible with 2.7.x.
If we have to make it compatible with 2.8.0 -- not an unreasonable suggestion -- we should keep the new directory an locate and copy existing dbs.  This seems like pretty big change to me.  Again, though, not unreasonable.  I think a better suggestion would be to get this out quickly and to advise people not to use 2.8.0.

Reply:
Agreed. We should make sure to put this into the release note.

Reply:
Is unlisting 2.8.0 an option?

Reply:
I don't know.  That is a discussion we need to have.  I don't think there is a way to be 100% backwards compatible.  The most common case for 2.8.1 is going to be that there are two dbs with identical names one in the root directory, one is .couchbase.  Deciding which to keep probably requires direct human intervention.